include/envoy/buffer/buffer.h

```diff
@@ -173,9 +174,9 @@ class Instance {
   /**
    * Write the buffer out to a file descriptor.
    * @param fd supplies the descriptor to write to.
-   * @return the number of bytes written or -1 if there was an error.
+   * @return a tuple with the number of bytes written or -1 if there was an error, and the errno.
```

Comment:
Here and above, I'd say "the errno, if any error occurred" or some other modification to make it clear that errno is only valid if the bytes_written part is -1

Reply:
Addressed in 36d2f52.